apps/addons/models.py

```diff
@@ -541,6 +541,8 @@ def get_url_path(self, more=False, add_prefix=True):
         # If more=True you get the link to the ajax'd middle chunk of the
         # detail page.
         view = 'addons.detail_more' if more else 'addons.detail'
+        if not self.is_listed:
+            view = 'devhub.addons.edit'
```

Comment:
We discussed that some more with Wil, and indeed, it's going to be confusing, so I'm going to change that to just not link at all if it's an unlisted addon, in all the places it's displaying this link.